versions.yaml

```diff
@@ -122,17 +122,20 @@ assets:
     url: "https://github.com/kata-containers/kata-containers/tools/osbuilder"
     architecture:
       aarch64:
-        name: "ubuntu"
-        version: "latest"
+        name: &default-image-name "ubuntu"
+        version: &default-image-version "latest"
       ppc64le:
-        name: "ubuntu"
-        version: "latest"
+        name: *default-image-name
+        version: *default-image-version
       s390x:
-        name: "ubuntu"
-        version: "latest"
+        name: *default-image-name
+        version: *default-image-version
       x86_64:
-        name: &default-image-name "ubuntu"
-        version: "latest"
+        name: *default-image-name
+        version: *default-image-version
+        tdx:
+          name: *default-image-name
+          version: *default-image-version
```

Comment:
I'm not sure why we're adding TDX here.

Reply:
This is a suggestion from @jepio, which makes sense if you follow the code path from [`install_image_tdx()`](https://github.com/kata-containers/kata-containers/pull/6839/commits/532755ce31d069dafaa1427c55757f0973381647#diff-4e9dffeafb07dc536a419fa1c12eda2ed1fba26d98f3987694a13fafecc4362dR181). This allows us to simply get the guest OS name/version [based on a "variant" field](https://github.com/kata-containers/kata-containers/pull/6839/commits/532755ce31d069dafaa1427c55757f0973381647#diff-4e9dffeafb07dc536a419fa1c12eda2ed1fba26d98f3987694a13fafecc4362dR169-R175) instead of having a bunch of [if statements in the image builder](https://github.com/kata-containers/kata-containers/pull/6839/commits/532755ce31d069dafaa1427c55757f0973381647#diff-c4e252149c7d3f7767fa98bf90ccef0e2cfe8a9a9065f08b1cb181df001ea0b8L113-L123).

Reply:
Makes sense.